packages/x-date-pickers/src/internals/components/PickersToolbar.tsx

```diff
@@ -129,7 +121,10 @@ export const PickersToolbar = React.forwardRef(function PickersToolbar<TDate, TV
           ownerState={ownerState}
           color="inherit"
           data-mui-test="toggle-mobile-keyboard-view"
-          aria-label={getMobileKeyboardInputViewButtonText(isMobileKeyboardViewOpen, viewType)}
+          aria-label={localeText.toolbarInputTypeSwitchingButtonAriaLabel(
```

Comment:
```suggestion
          aria-label={localeText.toolbarInputToViewSwitchButtonAriaLabel(
```

Right now I don't understand what the button is doing just with the locale key.
Mine is not great either.

Do we agree that this button is used to toggle the input inside the modal in mobile ?

Reply:
Yes, except that it is not necessary on mobile. 
With `showToolbar` prop you can let the toolbar (and so the button) appears on desktop too

Reply:
I tried with `toolbarInterfaceModifierButtonAriaLabel` but the real problem is this button is rarely used

Reply:
@m4theushw if you have an idea
All mine are crappy :laughing:

Reply:
@LukasTy Maybe you will have an idea about how to name this localisation key